### PR TITLE
permit to set virtual env options with variable VENV_OPTS

### PR DESCRIPTION
**Pull request by gdchamal**

### CODE REVIEW

File: caliopen.env.tmpl [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+#
+# Base url for git clone commands
+#
+BASE_URL="https://github.com/gdchamal"
+
+#
+# Virtualenv options
+#
+VENV_OPTS="--python=python2.7"
```
Comment (themouette) [inline caliopen.env.tmpl]: Won't it be forced for travis tests as well ?

Comment (themouette) [inline caliopen.env.tmpl]: What about checking pre-existence of a VIRTUALENV env variable?

**Comment by gdchamal** [comment]
It's template file update not real environment, so won't apply to travis.

**Comment by themouette** [comment]
Right, I missed that.

**Pull request merged**